lesson-7/script.js

```diff
@@ -0,0 +1,157 @@
+const isNumber = (number) => {
+    return typeof (number) === 'number' && !isNaN(number)
+}
+const isString = (string) => {
+    return typeof (string) === 'string' && !!string
```

Comment:
Also consider this check

```suggestion
    return typeof (string) === 'string' && string.length > 0
```